tutorials/deposit-withdraw.md

```diff
@@ -0,0 +1,293 @@
+Operation successfully injected in the node.
+Operation hash is 'ooFnJgW5jUsFuYFxt4ip3mXEcjjdytHUgaCUGe3PvvEhiMzoa65'
+Waiting for the operation to be included...
+Operation found in block: BKphQACo4G2y3hTXpKePWXF8sB7MEQJVmfWcU1VmURcvwa18f33 (pass: 3, offset: 0)
+This sequence of operations was run:
+  Manager signed operations:
+    From: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb
+    Fee to the baker: ꜩ0.000825
+    Expected counter: 12
+    Gas limit: 4725
+    Storage limit: 125 bytes
+    Balance updates:
+      tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb ........... -ꜩ0.000825
+      fees(tz1YPSCGWXwBdTncK2aCctSZAXWvGsGwVJqU,4) ... +ꜩ0.000825
+    Transaction:
+      Amount: ꜩ0
+      From: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb
+      To: KT1WyREzcYATwbcbGPnLvDvUAHQcY7SJLm82
+      Entrypoint: deposit
+      Parameter: (Pair (Pair 0x "KT1DNb3Hap4DwkpVmiYxH7UmcjmZANtQLEFv")
+                       "tz1RPNjHPWuM8ryS5LDttkHdM321t85dSqaf")
+      This transaction was successfully applied
+      Updated storage: {}
+      Storage size: 290 bytes
+      Consumed gas: 2808.124
+    Internal operations:
+      Transaction:
+        Amount: ꜩ0
+        From: KT1WyREzcYATwbcbGPnLvDvUAHQcY7SJLm82
+        To: KT1DNb3Hap4DwkpVmiYxH7UmcjmZANtQLEFv
+        Entrypoint: deposit
+        Parameter: (Pair 0x00003f0b3dd85deb3013c6a239fe9e3783ace4f8e37c
+                         (Pair 0x01f59b5e7986cec18cfc99b3a9c4a601b73dbdea1c00 (Pair 0x 1000000)))
+        This transaction was successfully applied
+        Updated storage:
+          (Pair (Pair (Pair (Pair 0xdf289a57e28aed1e42f84e65bb2c9e897746a16acb2f297295ecf3823ee37cc0 97)
+                            (Pair 0x0e5751c026e543b2e8ab2eb06099daa1d1e5df47778f7787faab45cdf12fe3a8
+                                  0x8e18383c67c247b222b328d73690f939b3ea05b5644a6f6f3bb460e0c463af68))
+                      { 0x00031958f137649fc62902c090e042c1fe03270b13 ;
+                        0x0012b1a66c13bce550ce285f8085506fca460544bc ;
+                        0x00021a0ead6b4e615981eac146b375f9040ef3c600 ;
+                        0x00f752ef4b5df8da5fef86fb5065cbc3eb241324f9 })
+                (Pair (Pair 4 5) 6))
+        Updated big_maps:
+          Set map(6)[(Pair 0x01f59b5e7986cec18cfc99b3a9c4a601b73dbdea1c00 0x)] to (Pair 0x01f59b5e7986cec18cfc99b3a9c4a601b73dbdea1c00 (Pair 0x 1000000))
+        Storage size: 4649 bytes
+        Paid storage size diff: 105 bytes
+        Consumed gas: 1816.734
+        Balance updates:
+          tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb ... -ꜩ0.02625
+
+The operation has only been included 0 blocks ago.
+We recommend to wait more.
+Use command
+  tezos-client wait for ooFnJgW5jUsFuYFxt4ip3mXEcjjdytHUgaCUGe3PvvEhiMzoa65 to be included --confirmations 5 --branch BM5WddbfGLMEwq26c8sF9u5qCsWqyfNAisMiQaYUgncEatnB5Mq
+and/or an external block explorer.
+```
+
+## Wallet creation
+
+Alice need an address on deku
+
+```bash
+deku-cli create-wallet
```

Comment:
I guess we should add an output example. This will make the following sentence (`and save the field address`) easier to understand.